astroid/brain/brain_typing.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2016 David Euresti <david@dropbox.com>
+
+"""Astroid hooks for typing.py support."""
+
+from astroid import (
+    MANAGER, UseInferenceDefault, extract_node, inference_tip,
+    nodes)
+
+def infer_typing_namedtuple(node, context=None):
+    """Infer a typing.NamedTuple(...) call."""
+    # This is essentially a namedtuple with different arguments
+    # so we extract the args and infer a named tuple.
+    func = next(node.func.infer())
+    if func.qname() != 'typing.NamedTuple':
+        raise UseInferenceDefault
+
+    if len(node.args) != 2:
+        raise UseInferenceDefault
+
+    typename = node.args[0].as_string()
+    names = []
+    try:
+        for elt in node.args[1].elts:
+            if len(elt.elts) != 2:
+                raise UseInferenceDefault
+            names.append(elt.elts[0].as_string())
+    except AttributeError:
```

Comment:
Well I'm not checking whether `node.args[1]` actually has an `elts` Or whether that elt has more elts in `elt.elts[0]`